src/sdk/service/network-validation-service/network-validation-service.ts

```diff
@@ -0,0 +1,468 @@
+import { IPv4AddressRange, IPv4NetworkDescriptor } from './network-validation-interface';
+
+/**
+ * Network Validation Service
+ */
+export abstract class NetworkValidationService {
+  /**
+   * Gets the integer representation of an IP address.
+   * @param {string} ip: the ip address.
+   * @return {number} The integer representation.
+   */
+  public static getIPv4AsInteger(ipAddress: string): number | null {
```

Comment:
This should be private since the representation of an IP as an integer is an implementation detail of the service and does not conform to any kind of specification.